eng/performance/scenarios.yml

```diff
@@ -93,6 +93,10 @@ jobs:
             displayName: Build SOD tool
             env:
               PERFLAB_TARGET_FRAMEWORKS: $(_Framework)
+          - script: $(CorrelationStaging)dotnet\dotnet publish -c Release -o $(CorrelationStaging)FailureReporter -f $(_Framework) -r win-${{parameters.architecture}} $(Build.SourcesDirectory)\src\tools\Reporting\FailureReporting\FailureReporting.csproj
+            displayName: Build FailureReporter tool
+            env:
+              PERFLAB_TARGET_FRAMEWORKS: $(_Framework)
```

Comment:
This also needs to be built on Linux (below.)

Reply:
How are the Linux build instructions distinct from the Windows ones?

Reply:
The `yml` files have branches everywhere for Linux and Windows to work around path construction issues. This change is in the Windows branch, so it needs to be mirrored in the Linux branch below (but with the correct path separators.)